configs/ddppo_pointnav.yaml

```diff
@@ -0,0 +1,62 @@
+BASE_TASK_CONFIG_PATH: "configs/challenge_pointnav2020.local.rgbd.yaml"
+TRAINER_NAME: "ddppo"
+ENV_NAME: "NavRLEnv"
+SIMULATOR_GPU_ID: 0
+TORCH_GPU_ID: 0
+VIDEO_OPTION: []
+TENSORBOARD_DIR: "tb"
+VIDEO_DIR: "video_dir"
+TEST_EPISODE_COUNT: 994
+EVAL_CKPT_PATH_DIR: "data/new_checkpoints"
+NUM_PROCESSES: 2
```

Comment:
This really should be at least 4 for 2 mini batches.  Only experience from 1 rollout can be really high-variance.  Changing the number of mini batches to 1 makes more sense if only 2 can be fit.